cluster/charts/rook-ceph-cluster/templates/ingress.yaml

```diff
@@ -20,7 +14,14 @@ spec:
         paths:
           - path: {{ .Values.ingress.dashboard.host.path | default "/" }}
             backend:
-{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
+{{- if (semverCompare "<1.18-0" (include "capabilities.kubeVersion" .)) }}
```

Comment:
@TomHellier In what version exactly is the new api available? We are seeing the [integration tests fail now on 1.18](https://github.com/rook/rook/runs/4369200614?check_suite_focus=true) since this change merged with the following error:
```
Error: unable to build kubernetes objects from release manifest: error validating "": 
error validating data: ValidationError(Ingress.spec.rules[0].http.paths[0].backend):
unknown field "service" in io.k8s.api.networking.v1beta1.IngressBackend , err -> exit status 1
```

Reply:
Hey,

https://github.com/bitnami/charts/blob/8150149f0bb746e86ff0029fc375d43775bdf15a/bitnami/common/templates/_ingress.tpl#L37 is where I took inspiration from, are we now deploying against all versions of k8s in the integration tests?

Ill have a look this evening in more detail

Reply:
Ok thanks for taking a look. In master/release branches, the rook integration tests run on more versions of k8s to give us broader coverage like this.